modules/dnn/test/test_onnx_conformance.cpp

```diff
@@ -1011,54 +1019,116 @@ class Test_ONNX_conformance : public TestWithParam<ONNXConfParams>
         }
         return hasFallbacks;
     }
-};
 
-TEST_P(Test_ONNX_conformance, Layer_Test)
-{
-    std::string name = test_case.name;
-    //Backend backend = ...;
-    //Target target = ...;
+    static void SetUpTestCase()
+    {
+        parser_black_list = {
```

Comment:
I fixed this. But looking at resulting code... do we need includes here? I think it would be better to move these filter lists to extra and just read them.